pyalgostrategypool/options_bear_call_ladder_v2.py

```diff
@@ -0,0 +1,130 @@
+from constants import *
+from strategy.core.strategy_options_base_v2 import StrategyOptionsBaseV2
+
+
+class StrategyOptionsBearCallLadder(StrategyOptionsBaseV2):
+    name = 'Options Bear Call Ladder Template v2'
```

Comment:
rename all files and rename all names to remove V2 later once everything is finalized
do this for all strategies named v2

Reply:
Not Done